Sources/PackageDescription/SupportedPlatforms.swift

```diff
@@ -575,6 +580,14 @@ extension SupportedPlatform {
         @available(_PackageDescription, introduced: 5.7)
         public static let v22: DriverKitVersion = .init(string: "22.0")
     }
+
+    public struct CustomPlatformVersion: AppleOSVersion {
+        static var name: String = "custom platform"
+        static var minimumMajorVersion = 0
+
+        fileprivate init(uncheckedVersion version: String) {
```

Comment:
no need to implement this?

Reply:
Not needed at this time since we are only using `CustomPlatformVersion` for the validation (technically even only to get the right string into the diagnostic, otherwise we could just use one of the other platform type).